app/views/shared/vote/_static_icons.html.erb

```diff
@@ -1,3 +1,3 @@
-<i class="fa fa-chevron-circle-up not-clickable"></i>
-<i class="fa fa-chevron-circle-down not-clickable"></i>
-<i class="fa fa-times-circle not-clickable"></i>
+<%= tag.i class: "fa fa-chevron-circle-up not-clickable" %>
+<%= tag.i class: "fa fa-chevron-circle-down not-clickable" %>
+<%= tag.i class: "fa fa-times-circle not-clickable" %>
```

Comment:
I'd revert this. I don't see an advantage of pulling Rails in to essentially render the markup that was here previously.